lib/imgur.js

```diff
@@ -105,15 +109,13 @@ imgur._imgurRequest = async (operation, payload, extraFormParams) => {
     };
   }
 
-  if (operation === 'upload' || operation === 'update') {
+  if (typeof extraFormParams === 'object') {
```

Comment:
Instead of limiting this block of code by the type of operation, let's just add the form params if the extraFormParams parameter is an object. This includes the 'upload' and 'update' operations as well as anything else that would need to use extraFormParams in an imgurRequest.

Reply:
🎉